Modules/booking/Model/BkGraph.php

```diff
@@ -531,7 +532,7 @@ public function getCamembertContent($id_space, $month_start, $year_start, $month
             $numMachinesFormes[$i][0] = $mFL[0];
             $numMachinesFormes[$i][1] = $req->rowCount();
 
-            $curentAngle = 2 * pi() * $numMachinesFormes[$i][1] / $numTotal;
+            $curentAngle = 2 * pi() * $numMachinesFormes[$i][1] / $numMachinesFormesTotal;
```

Comment:
saw that just passing by. I think there was a change of variable name, but not applied to this occurrence